packages/frontend/app/components/curriculum-inventory/sequence-block-overview.hbs

```diff
@@ -1,498 +1,512 @@
       <div class="title" data-test-title>{{t "general.overview"}}</div>
       <div class="curriculum-inventory-sequence-block-overview-content">
-        {{#if (or this.load.isRunning this.reload.isRunning)}}
-          <div class="block"><LoadingSpinner /></div>
-        {{else}}
-          <div class="block course" data-test-course>
-            <span>
-              <label for="course-{{templateId}}">{{t "general.course"}}:</label>
-              {{#if @canUpdate}}
-                <EditableField
-                  @value={{this.course.title}}
-                  @save={{this.saveCourse}}
-                  @close={{this.revertCourseChanges}}
-                  @clickPrompt={{t "general.selectCourse"}}
-                >
-                  <select id="course-{{templateId}}" {{on "change" this.updateCourse}}>
-                    <option value="" selected={{is-empty this.course}}>{{t
-                        "general.selectCourse"
-                      }}</option>
-                    {{#each (sort-by "title" this.linkableCourses) as |obj|}}
-                      <option
-                        value={{obj.id}}
-                        selected={{eq obj this.course}}
-                      >{{obj.title}}</option>
-                    {{/each}}
-                  </select>
-                </EditableField>
-              {{else}}
-                <span data-test-course-title>{{if
-                    this.course
-                    this.course.title
-                    (t "general.notApplicableAbbr")
-                  }}</span>
-              {{/if}}
-            </span>
-            {{#if this.course}}
+        <div class="block course" data-test-course>
+          <span>
+            <label for="course-{{templateId}}">{{t "general.course"}}:</label>
+            {{#if @canUpdate}}
+              <EditableField
+                @value={{this.course.title}}
+                @save={{this.saveCourse}}
+                @close={{this.revertCourseChanges}}
+                @clickPrompt={{t "general.selectCourse"}}
+              >
+                <select id="course-{{templateId}}" {{on "change" this.updateCourse}}>
+                  <option value="" selected={{is-empty this.course}}>{{t
+                      "general.selectCourse"
+                    }}</option>
+                  {{#each (sort-by "title" this.linkableCourses) as |obj|}}
+                    <option value={{obj.id}} selected={{eq obj this.course}}>{{obj.title}}</option>
+                  {{/each}}
+                </select>
+              </EditableField>
+            {{else}}
+              <span data-test-course-title>{{if
+                  this.course
+                  this.course.title
+                  (t "general.notApplicableAbbr")
+                }}</span>
+            {{/if}}
+          </span>
+          {{#if this.selectedCourse}}
+            {{#if this.selectedCourse.id}}
```

Comment:
If there is a `selectedCourse`, but no `selectedCourse.id`, then nothing appears?

Reply:
that's correct. if the user un-selects a previously attached course, i need to track this somehow. i'm using an empty object for this `{}`. this has to be different than no user selection, which is `null`.  triple state - empty object, null, or course model.
not sure how to do this otherwise, i made several passes over this. please let me know if you have a better idea here.